code/modules/surgery/organ_manipulation.dm

```diff
@@ -19,25 +19,7 @@
 		/datum/surgery_step/retract_skin,
 		/datum/surgery_step/clamp_bleeders,
 		/datum/surgery_step/incise,
-		/datum/surgery_step/manipulate_organs/internal,
-		/datum/surgery_step/close,
-	)
-
-/datum/surgery/organ_manipulation/external
-	name = "Feature manipulation"
```

Comment:
I do think this is a bad change, the point is to let people easily change cosmetic organs.  It's fine for now, at least it's one less thing that can break and it can be readded later if demand is there

Reply:
There is not a foolproof way for me to identify "external" organs and therefore it's hard for me to keep this in.

Reply:
Also this is literally the same step, you can't even like, restyle the organs (?)

Reply:
Nah it's way shorter than normal organ manip